src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -194,7 +194,11 @@ int fmi3_xml_variable_is_array(fmi3_xml_variable_t* v) {
 }
 
 int fmi3_xml_get_variable_has_start(fmi3_xml_variable_t* v) {
-    return (v->type->structKind == fmi3_xml_type_struct_enu_start);
+    if (v->type) {
```

Comment:
I'm not surprised. We really need to decide how we want to handle this.

Can you add a comment `// Can be null if parse error`? It's not intuitive why the check is there otherwise.

Reply:
Added the comment.